public/js/app.js

```diff
@@ -159,7 +159,7 @@ async function main(window, document) {
     for (i = 0; i < mapping.length; i++) {
       if (mapping[i].id === 'user-picture' && mapping[i].data) {
         document.getElementById(mapping[i].id).src = mapping[i].data;
-      } else if (mapping[i].data) {
+      } else {
```

Comment:
This keeps the locks when the users aren't friends, but removes them if they are friends